lib/feature_management.rb

```diff
@@ -118,11 +118,14 @@ def self.log_to_stdout?
   end
 
   def self.phone_recaptcha_enabled?
-    IdentityConfig.store.recaptcha_site_key_v2.present? &&
-      IdentityConfig.store.recaptcha_site_key_v3.present? &&
+    return false if IdentityConfig.store.recaptcha_site_key_v2.blank? ||
+                    IdentityConfig.store.recaptcha_site_key_v3.blank? ||
```

Comment:
we need both a v2 and a v3 key?

Reply:
Yes, the current implementation uses v3 for the initial interaction, and then redirects to a checkbox (v2) as an error fallback if they fail that first check.